app/Domains/Vault/ManageVault/Api/Controllers/VaultController.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace App\Domains\Vault\ManageVault\Api\Controllers;
+
+use App\Domains\Vault\ManageVault\Services\CreateVault;
+use App\Domains\Vault\ManageVault\Services\DestroyVault;
+use App\Domains\Vault\ManageVault\Services\UpdateVault;
+use App\Http\Controllers\ApiController;
+use App\Http\Resources\VaultResource;
+use App\Models\Vault;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Database\QueryException;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class VaultController extends ApiController
```

Comment:
We can add the group here
```suggestion
/**
 * @group Vault management
 * @subgroup Vaults
 */
class VaultController extends ApiController
```